samples/chemistry/PythonIntegration/h2.yaml

```diff
@@ -7,14 +7,14 @@ format: {version: '0.1'}
 generator: {source: nwchem, version: '6.8'}
 integral_sets:
 - basis_set: {name: sto-3g, type: gaussian}
-  coulomb_repulsion: {units: hartree, value: 0.713776188}
+  coulomb_repulsion: {units: hartree, value: 0.32584805972887831}
   energy_offset: {units: hartree, value: 0.0}
   fci_energy: {lower: 0.0, units: hartree, upper: 0.0, value: 0.0}
   geometry:
     atoms:
     - coords: [0.0, 0.0, 0.0]
       name: H
-    - coords: [0.0, 0.0, 1.624]
+    - coords: [0.0, 0.0, 0.859]
```

Comment:
It looks like the bond length has been changed in this PR as well. That should be perfectly fine since there's no particular "right" bond length to use, but out of curiousity, do we know if the old Coulomb repulsion and SCF energy were consistent for the old bond length?

Reply:
The bond length is the same, I just made it consistent with the units (angstrom).